Boolean.CSharp.Main/Core.cs

```diff
@@ -1,14 +1,122 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Text;
+using System.Transactions;
+using static System.Runtime.InteropServices.JavaScript.JSType;
 
 
 namespace Boolean.CSharp.Main
 {
-    public class Core
+    
+    public interface Iaccount
     {
+        public List<Transaction> GetTransactions();
+        public bool AddTransaction(float f, DateTime? date );
+        public float GetBalance();
+        string GenerateTransationsHistory();
+    }
+    public class Account : Iaccount
+    {
+        private List<Transaction> transactions;
+        public  List<Transaction> Transactions { get { return transactions; } }
+    
+        public Account() {
+
+
+
+            transactions = new List<Transaction>() ;
+        }
+
+        
+
```

Comment:
code is messy, a lot of empty spaces, inconsistent indentations..., very hard to read